mix.exs

```diff
@@ -47,6 +47,7 @@ defmodule Membrane.FFmpeg.Transcoder.MixProject do
       {:membrane_mpeg_ts_plugin, "~> 2.0"},
       {:membrane_text_format, "~> 1.0"},
       {:kim_subtitle, "~> 0.1.2"},
+      {:membrane_subtitles_plugin, "~> 0.1.0"},
```

Comment:
Remove all subtitles related things, maybe just keep the text format to output something that is not plain RemoteStream, but that might be an idea as well.

Reply:
I didn't do that because it was quite strange to emit SRT cues. In case we change it to a custom dvb teletext parser it would be a breaking change. So my idea was to just have simple text buffers with pts and dts. Without the SRT parsing we will only have buffers with text (no timing at all).